web-ui/src/main/resources/catalog/components/common/map/mapService.js

```diff
@@ -845,12 +848,24 @@
                       gnUrlUtils.toKeyValue({
                         service: 'WFS',
                         request: 'GetFeature',
-                        version: '1.1.0',
+                        version: getCapLayer.version,
                         srsName: map.getView().getProjection().getCode(),
                         bbox: extent.join(','),
                         typename: getCapLayer.name.prefix + ':' +
                                    getCapLayer.name.localPart}));
                   
+                  //Fix, ArcGIS fails if there is a bbox:
```

Comment:
This is the "ugly" patch. I couldn't understand why ArcGIS failed on bbox request. But while ArcGIS fix this, if we want to show their layers, we will have to accomodate.